.github/workflows/migrations-check.yml

```
@@ -1,19 +1,18 @@
 name: Migrations check
 
 on:
-  workflow_dispatch:
-  pull_request:
   push:
-    branches:
-      - master
+    branches: [master]
+  pull_request:
+    branches: [master]
```

We want this check to run against all Pull Requests but now it'll only run against master branch, so it should be updated. 
```suggestion
    branches: "**"
```
or
```suggestion

```
both would work for this case to run the check against the pull requests of all branches.